redwood-flex-container/src/commonMain/kotlin/app/cash/redwood/flexcontainer/FlexContainer.kt

```diff
@@ -613,13 +610,16 @@ public class FlexContainer {
             flexItem = child,
             padding = flexLine.sumCrossSizeBefore,
           )
-          val childHeightMeasureSpec = MeasureSpec.from(newHeight, MeasureSpecMode.Exactly)
-          child.applyMeasure(childWidthMeasureSpec, childHeightMeasureSpec)
-          childMeasuredWidth = child.measuredWidth
-          childMeasuredHeight = child.measuredHeight
+          child.measuredWidth = when (childWidthMeasureSpec.mode) {
+            MeasureSpecMode.Exactly -> childWidthMeasureSpec.size
+            MeasureSpecMode.AtMost -> child.measurable.width(newHeight).coerceAtMost(childWidthMeasureSpec.size)
+            MeasureSpecMode.Unspecified -> child.measurable.width(newHeight)
+            else -> throw AssertionError()
+          }
+          child.measuredHeight = newHeight
```

Comment:
This changes the logic slightly to avoid a second measure pass (which Compose doesn't allow).